examples/messaging/eth-config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "provider": {
+    "rpc_endpoint": "https://eth.merkle.io",
```

Comment:
how will a node specify an rpc endpoint with their api key ? is that config hidden?

Reply:
yes, only the person running the node will have access to it. we can create an issue though to fetch values from the ENV if that's better?

Reply:
probably yes